package.json

```diff
@@ -29,7 +29,13 @@
     "start": "nodemon --exec babel-node ./src/app.js",
```

Comment:
install cross-env and use it to set development test and production environments. This line should be "start:dev"